server/api/src/rsFileChksum.cpp

```diff
@@ -190,14 +189,21 @@ int fileChksum(rsComm_t* rsComm,
         irods::getHasher( irods::MD5_NAME, hasher );
     }
 
+    // get the buffer size from server configuration - default to 1 MiB
+    int read_buffer_size;
+    try {
+        read_buffer_size = irods::get_advanced_setting<int>("checksum_read_buffer_size_in_bytes");
+    }
+    catch (const irods::exception&) {
+        rodsLog(LOG_DEBUG, "%s :: checksum_read_buffer_size_in_bytes not set.  Using default of 1 MB.", __func__);
```

Comment:
Replace use of `rodsLog` with new logger.

You can add the following near the top of the file.
```cpp
#include "irods/irods_logger.hpp"

using log_api = irods::experimental::log::api;
```

And then replace `rodsLog` with `log_api::debug`.

Reply:
done